.gitlab-ci.yml

```diff
@@ -68,73 +68,78 @@ init_workspace:
     - chmod 700 ~/.ssh
     - ssh-keyscan github.com >> ~/.ssh/known_hosts
     # Clone all repos in WORKSPACE
+    - cd /tmp
```

Comment:
Is this necessary? I think you're using absolute paths everywhere.

Reply:
Yep. This was before I changed WORKSPACE definition to be an absolute path. I'll clean it up